sap-s4hana-cloud/2.4/modules/ROOT/pages/sap-s4hana-cloud-connector-reference.adoc

```diff
@@ -53,6 +53,26 @@ Basic authentication is the simplest way of enforcing access controls because it
 |===
 | Name | Type | Description | Default Value | Required
 | Service base URL a| String |  The base URL of the SAP S/4HANA instance to access from the connector and from which all other endpoints are derived. |  | x
+| Cache Entry TTL a| Number |  The time to live. The time a cache entry is kept in the cache before being evicted. A value of 0 disables the cache. |  0 |
+| Cache Entry TTL Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit a cache entry will be kept in the cache until evicted. |  MINUTES |
```

Comment:
Usually for our reference guides, when we have a parameter and it has an equivalent parameter for its "Time Unit", for the description we can just say "Time unit for the * Cache Entry TTL * field."

Reply:
Much better! Fixed.